templates/templates/repo/scripts/module.mk

```diff
@@ -22,8 +26,14 @@ check: lint check-docs ## run all checks on this module
 lint: lint-tf check-docs ## run all linters on this module
 .PHONY: lint
 
-lint-tf: terraform ## run terraform linters on this module
+lint-tf: setup terraform ## run terraform linters on this module
```

Comment:
setup is called first so that tflint is installed if the plugin is configured properly